examples/hotel-list.js

```diff
@@ -2,25 +2,21 @@ var expedia = require("../lib/expedia")();
 
 // a complete list of options is available at http://developer.ean.com/docs/hotel-list/
 var options = {
-  "customerSessionId" : "thisisauniqueID",
   "customerIpAddress" : "127.0.0.1",
-  "customerUserAgent" : "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_8_4) AppleWebKit/537.36 (KHTML, like Gecko)",
+  "customerUserAgent" : "Node.js",
   "HotelListRequest": {
```

Comment:
This contains the user agent, not sure if we can use Node.js. We can use the version `TravelBank/{api_version}`
 http://dev.ean.com/docs/common. I will implement the dynamic part.

Reply:
@oldmonk7 how can we get the api_version?

Reply:
I think user agent is just a string, it works with 'Node.js' as well.

Reply:
We will figure out, where can we store the api-version, can be as simple as env var.
Meanwhile, you can keep it as `Node.js`. Trivial enough.